agent/host_agent_test.go

```diff
@@ -8,6 +8,7 @@ import (
 	"context"
 	"fmt"
 	"io/ioutil"
+	restclient "k8s.io/client-go/rest"
```

Comment:
Can we move this import from line 11 to somewhere between line 19-34

Reply:
The restclient import was present in a different import group separated by a blank line, perhaps that's why goimports did not take care in this case.